app/src/lib/git/core.ts

```diff
@@ -4,6 +4,7 @@ import {
   GitError as DugiteError,
   IGitExecutionOptions as DugiteExecutionOptions,
 } from 'dugite'
+import { GitErrorRegexes as DugiteErrorRegexes } from 'dugite/build/lib/errors'
```

Comment:
Not super happy about this, I noticed `GitErrorRegexes` wasn't properly exposed in `dugite` and instead had to look for it here 😕